.github/workflows/hotfix-release.yml

```
@@ -0,0 +1,230 @@
+
+jobs:
+  hotfix:
+    name: Hotfix Release
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: "CopilotKit"
+    timeout-minutes: 15
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)